src/app/info/feedback/feedback-form/feedback-form.component.ts

```diff
@@ -9,7 +9,7 @@ import { TranslateService } from '@ngx-translate/core';
 import { AuthService } from '../../../core/auth/auth.service';
 import { EPerson } from '../../../core/eperson/models/eperson.model';
 import { getFirstCompletedRemoteData } from '../../../core/shared/operators';
-import { Router } from '@angular/router';
+import { Router, UrlSegmentGroup } from '@angular/router';
```

Comment:
@atarix83 or @rezartatis : Tiny thing, looks like you added an unnecessary import here, as LGTM noted.  This change can be removed entirely, as you don't use `UrlSegmentGroup` in the code.

I'll retest this later today to make sure everything works, but I'd appreciate it if we can quickly fix this line since we received an alert from LGTM